app/src/androidTest/java/ch/epfl/sweng/runpharaa/SettingsActivityTest.java

```diff
@@ -57,6 +58,16 @@ public void init() {
         r = InstrumentationRegistry.getTargetContext().getResources();
     }
 
+    @Test
+    public void deleteUserTest(){
```

Comment:
As stated previously, we should always check something in the tests and not only go through the code